content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -158,6 +158,41 @@ and
 Note that the information reported as Pod status depends on the current
 [ContainerState](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#containerstatus-v1-core).
 
+## Container States
+
+Once pod is assigned to a node by scheduler, kubelet starts creating container using container runtime.There are three possible states of containers, i.e Waiting, Running and Terminated. To check state of container, you can use `kubectl describe pod` command. State will be displayed for each container within that pod.
```

Comment:
Change to:"

"Once a Pod is assigned to a node by the scheduler, kubelet starts creating containers using the container runtime. There are three possible states of containers: Waiting, Running, and Terminated. To check the state of a container, you can run `kubectl describe pod [POD_NAME]`. State is displayed for each container within that Pod."

Reply:
Suggested changes are committed. Please check